server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/ExportAction.java

```diff
@@ -101,23 +99,28 @@ public void define(WebService.NewController controller) {
         // This deprecated key is only there to be able to deal with redirection from /profiles/export
         .setDeprecatedKey("format", "6.3");
     }
+
+    wsSupport.createOrganizationParam(action).setSince("6.4");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)